examples/buildpacks/README.md

```diff
@@ -37,11 +37,11 @@ tkn tr describe --last -o jsonpath='{.metadata.annotations.chains\.tekton\.dev/s
 # Should output "true"
 
 # Export the value of IMAGE_URL from the last pipeline run and the associated taskrun name:
-export IMAGE_URL=$(tkn pr describe --last -o jsonpath='{..taskResults}' | jq -r '.[] | select(.name | match("IMAGE_URL$")) | .value')
-export TASK_RUN=$(tkn pr describe --last -o json | jq -r '.status.taskRuns | keys[] as $k | {"k": $k, "v": .[$k]} | select(.v.status.taskResults[]?.name | match("IMAGE_URL$")) | .k')
-
-## If using the registry-proxy
-# export IMAGE_URL="$(echo "${IMAGE_URL}" | sed 's#'${REGISTRY}'#127.0.0.1:5000#')"
+IMAGE_URL=$(tkn pr describe --last -o jsonpath='{..taskResults}' | jq -r '.[] | select(.name | match("IMAGE_URL$")) | .value')
+TASK_RUN=$(tkn pr describe --last -o json | jq -r '.status.taskRuns | keys[] as $k | {"k": $k, "v": .[$k]} | select(.v.status.taskResults[]?.name | match("IMAGE_URL$")) | .k')
+if [ "${REGISTRY}" = "registry.registry" ]; then
```

Comment:
seems like this will be confusing for readers. I understand that CI is using the internal registry and that is why this is done. Is the plan for the examples to use this also? Seems like they are still pointing to `ttl.sh`.

Reply:
The default is `ttl.sh`, but if you are using the built in registry, this avoids needing to copy the lines from inside a comment. The `if` gets skipped for any other registry. I've updated the PR to make it more clear how to use the local registry.